### PR TITLE
Initial testing of i18n code

### PR DESCRIPTION
**Pull request by pstaabp**
This checks in the i18n code and allows dynamic language changes.  To change the language, select the new language under User Settings (dropdown from Logged in User name).

### CODE REVIEW

File: src/boot/i18n.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { boot } from 'quasar/wrappers';
+import { createI18n } from 'vue-i18n';
+import messages from 'src/i18n';
+
+const i18n = createI18n({
+	locale: 'fr',
```
Comment (drgrice1) [inline src/boot/i18n.ts:6]: The default here should be 'en_US'.

File: src/layouts/MenuBar.vue [hunk 2]
```diff
@@ -1,43 +1,49 @@
...
+
+			<q-card-section class="q-pt-none">
+				<q-select label="Language" v-model="$i18n.locale" :options="$i18n.availableLocales" />
+			</q-card-section>
+			<q-card-actions align="right" class="bg-white text-teal">
+				<q-btn flat label="OK" v-close-popup />
+			</q-card-actions>
+		</q-card>
+	</q-dialog>
+	</div>
```
Comment (drgrice1) [inline src/layouts/MenuBar.vue:46]: This completely reverts the addition of the WeBWorK and MAA logos.  It looks like you forgot to pull before starting work.  Either that or you forgot to relegate your work with the upstream changes.

File: src/layouts/MenuBar.vue [hunk 3]
```diff
@@ -51,7 +57,7 @@ export default defineComponent({
 	emits: {
 		toggle: null
 	},
```
Comment (drgrice1) [inline src/layouts/MenuBar.vue:59]: See there are no other events that need validation, the shorthand array form is preferable.
```suggestion
	emits: ["toggle"],
```

**Review by drgrice1 — APPROVED** [review]
Looks good.  But there are a few issues.  I will submit a pull request to this that adds back the logos and addresses my other comments.

File: src/layouts/MenuBar.vue (hunk 2 above)
Comment (pstaabp) [reply src/layouts/MenuBar.vue:56]: I thought I had before doing this work.  Oops

File: src/boot/i18n.ts (hunk 1 above)
Comment (pstaabp) [reply src/boot/i18n.ts]: agreed.

**Pull request merged**